packages/api-explorer/src/Example.jsx

```diff
@@ -14,7 +14,68 @@ const Oas = require('./lib/Oas');
 
 const { Operation } = Oas;
 
-function Example({ operation, result, oas, selected, setExampleTab, exampleResponses }) {
+function isDisplayable(example, responseTypeCopy) {
+  if (!responseTypeCopy) return true;
+
+  return example.label === responseTypeCopy;
+}
+
+function getReactJson(example, responseTypeCopy) {
+  return (
+    <ReactJson
+      src={JSON.parse(example.code)}
+      collapsed={2}
+      collapseStringsAfterLength={100}
+      enableClipboard={false}
+      theme="tomorrow"
+      name={null}
+      displayDataTypes={false}
+      displayObjectSize={false}
+      key={example.code}
+      style={{
+        padding: '20px 10px',
+        backgroundColor: 'transparent',
+        display: isDisplayable(example, responseTypeCopy) ? 'block' : 'none',
+        fontSize: '12px',
+      }}
+    />
+  );
+}
+
+function showExamples(examples, setResponseType, responseType) {
+  let responseTypeCopy = responseType;
+  if (!responseTypeCopy && examples[0]) responseTypeCopy = examples[0].label;
+
+  return (
+    <div>
+      <select
+        style={{ backgroundColor: '#3d434a' }}
```

Comment:
It would probably be best if this were a class name instead. Can you add a class for this into `example/bundle-hub2.css`? I'll make sure we do the same to our production CSS.

Reply:
I've added this